modules/phase_field/doc/content/source/ics/PolycrystalVoronoiVoidIC.md

```diff
@@ -1,12 +1,12 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # PolycrystalVoronoiVoidIC
 
-!alert construction title=Undocumented Class
-The PolycrystalVoronoiVoidIC has not been documented, if you would like to contribute to MOOSE by
-writing documentation, please see [/generate.md]. The content contained on this page explains
-the typical documentation associated with a MooseObject; however, what is contained is ultimately
-determined by what is necessary to make the documentation clear for users.
+The PolycrystalVoronoiVoidIC generates a Voronoi tesslation to produce a grain structure with voids randomly distributed along the grain boundaries. The centroids of grains can be either generated from a set of random points or assigned from a file. It requires the number of voids to be greater than zero. In general, you should use [PolycrystalVoronoi](/PolycrystalVoronoi.md) to represent Voronoi grain structures without voids.
```

Comment:
Is there a check in this IC that the void radius will fit within the computational boundary?

Reply:
It will check if the center is within the computational domain.

Reply:
Okay, maybe a good idea to add a note to that effect then (that it checks the center but not the radius).  I know there was a student of David Bai who just ran into that issue.

Reply:
I just tried an input file with a large radius. The code is still running and the result still looks good. The only issue is that the bubbles might be overlapping.

Reply:
ok, then no worries.